includes/class-amp-theme-support.php

```diff
@@ -1043,6 +1043,10 @@ public static function prepare_response( $response, $args = array() ) {
 				1
 			);
 		}
+
+		// Move anything after </html>, such as Query Monitor output added at shutdown, to be moved before </body>.
+		$response = preg_replace( '#(</body>.*</html>)(.+)#s', '$2$1', $response );
```

Comment:
Warning: If there is something like `<!-- </body></html> -->` in the `body` then this will break. It would be better to use the DOM to move the content since we're parsing it below anyway. We can just move any siblings after the following siblings of `$dom->documentElement` to be added under the `body` element.